javascript/javascript-quiz.md

```diff
@@ -2023,6 +2023,9 @@ console.log(obj1);
 - [ ] intervalOver
 - [ ] None of the above
 
+[Reference](https://www.interviewbit.com/javascript-mcq/)
```

Comment:
```suggestion
[Reference](https://www.interviewbit.com/javascript-mcq/)
```

@ASAlishaa thanks for reference but it is hard to find the answer on this Web Site, do you have anything else?